platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/style/CircleLayerActivity.java

```diff
@@ -58,54 +60,97 @@ public void onCreate(Bundle savedInstanceState) {
       @Override
       public void onMapReady(@NonNull final MapboxMap map) {
         mapboxMap = map;
+
+        // add source
```

Comment:
What do you think about extracting this block of code into a `private` well named method and remove the comment?